From fe71a9ed1691901a22973386c2f0d7c1513f1433 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jaime=20Marqui=CC=81nez=20Ferra=CC=81ndiz?=
 <jaime.marquinez.ferrandiz@fastmail.net>
Date: Sat, 19 Feb 2022 21:00:13 +0100
Subject: [PATCH] cmake: Install hlc_main.c with hl.h and hlc.h
Upstream: submitted
References: gh#HaxeFoundation/hashlink#517

This matches the behaviour of the Makefile
---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 01c0014..d66d26a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -327,6 +327,10 @@ install(
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )
+install(
+    FILES src/hlc_main.c
+    TYPE INCLUDE
+)
 
 # uninstall target
 # https://gitlab.kitware.com/cmake/community/wikis/FAQ#can-i-do-make-uninstall-with-cmake
-- 
2.35.1

